nrf_modem/doc/gnss_interface.rst

```diff
@@ -476,6 +476,30 @@ In cases where GNSS is not running continuously, it may be beneficial to change
 
    1PPS is only supported by modem firmware v1.3.0 or later.
 
+Resolving the UTC time of 1PPS pulse occurrence
+===============================================
+
+As the next pulse time (beginning of an UTC second) is calculated from the previous valid PVT fix, the latest PVT fix notification needs to be used when resolving the UTC time of the 1PPS pulse.
+While the 1PPS pulse does not have any time delay, the PVT fix notification will always have some delay, both from the PVT solution calculation (approximately 100ms) and the notification message delivery.
+Therefore, the 1PPS pulse may come before the PVT notification that was used to calculate the exact pulse time.
+
+The UTC time of the 1PPS pulse can be calculated using the following formula:
+
+.. math::
+
+   round\ to\ nearest\ second\ (t_{(PVT,GPST)}+∆t+100 ms)
+
+t\ :sub:`(PVT,GPST)` \ is the GPS time stamp in the previous PVT notification and |delta| t is the time difference between the 1PPS pulse and the reception of previous PVT notification:
+
+.. |delta| unicode:: 0x394 .. capital delta sign
+   :rtrim:
+
+.. math::
+
+   ∆t=t_P-t_{PVT}
+
+Thus, |delta| t is always positive.
```

Comment:
```suggestion
Thus, |delta| ``t`` is always positive.
```

Reply:
@peknis This gets rendered like this, is this what we want?

![image](https://user-images.githubusercontent.com/2588548/211296046-c29a8ee4-1f22-450b-be56-917875d8f6e2.png)

Reply:
I thought we might want "t" to stand out. If it bothers you, take it out. It is not code, after all.

Reply:
I think it looks better if both `delta` and `t` are the same color and font. I tried the one below, but that gets rendered like this:
![image](https://user-images.githubusercontent.com/2588548/211302430-942a9a1e-fcba-4b57-b4eb-3eb961c7f677.png)
```suggestion
Thus, ``|delta| t`` is always positive.
```
So maybe we'll go with the original...